src/routes/living-form/containers/NeuburgForm.js

```diff
@@ -0,0 +1,219 @@
+                                                value={key} /></NarrowCol>
+          ))}
+        </Row>
+        <h3>Gültigkeitsdauer und Datenschutz</h3>
+        <Row>
+          <StdCol>
+            <SelectInput name='duration' label='Gültigkeitsdauer des Angebots' validations={[required]}>
+              <MenuItem value={3}>3 Tage</MenuItem>
+              <MenuItem value={7}>7 Tage</MenuItem>
+              <MenuItem value={14}>14 Tage</MenuItem>
+              <MenuItem value={30}>30 Tage</MenuItem>
+            </SelectInput>
+          </StdCol>
+        </Row>
+        <Row>
+          <WideCol>
+            <SingleCheckbox name='agreedToDataProtection' label='Ich akzeptiere die Datenschutzerklärung:'
+                            validations={[needsAcceptance]} />
+            <LawParagraph>
+              Ich willige ein, dass der Landkreis Neuburg-Schrobenhausen und die Tür an
+              Tür - Digital Factory gGmbH meine personenbezogenen Daten zum Zwecke
+              der Wohnraumakquise für anerkannte Flüchtlinge und bleibeberechtigte
+              Migranten erheben, verarbeiten und nutzen. Der Zweck ist ausschließlich
+              auf die Bearbeitung meines Mietangebots beschränkt.
+            </LawParagraph>
+            <LawParagraph>
+              Hierzu stellt die Tür an Tür - Digital Factory gGmbH meine
+              personenbezogenen Daten auf die öffentliche App Integreat, die vom
+              Landkreis Neuburg-Schrobenhausen genutzt wird, um Wohnungen und
+              Mietobjekte zielgerichtet zu vermitteln.
+            </LawParagraph>
+            <LawParagraph>
+              Die Einwilligung kann verweigert bzw. jederzeit ohne Angabe von Gründen über den in der Bestätigungsmail
+              versendeten Link zur dauerhaften Löschung widerrufen werden. In diesem
+              Fall erfolgt keine Bearbeitung meines Mietangebots; alle gespeicherten
+              personenbezogenen Daten werden gelöscht.
+            </LawParagraph>
+          </WideCol>
+        </Row>
+        <Row style={{margin: '20px 0', justifyContent: 'center'}}>
+          <Col center='true'>
+            <FormControl>
+              <SubmitButton validateAll={this.validateAll} type='submit' disabled={this.props.sending}>
+                {this.props.sending ? 'Wird gesendet...' : 'Mietangebot Senden'}
+              </SubmitButton>
+              {this.props.serverError && <FormHelperText error>{this.props.serverError}</FormHelperText>}
+            </FormControl>
+          </Col>
+        </Row>
+        <Row>
+          <StdCol>
+            Dieses Projekt wird aus Mitteln des Bayerischen Staatsministeriums für Arbeit und Soziales, Familie und
+            Integration gefördert.
+          </StdCol>
+          <StdCol>
+            <img src={staatLogo} style={{width: '100%'}} />
+          </StdCol>
+        </Row>
+      </Grid>
+    </Form>
+  }
+
+  handleSubmit = event => {
```

Comment:
This function looks a bit dirty imo :/
It kind of filters all of the arrays for null values. Why are there null values in them?

Reply:
If you have multiple inputs with the same name `name` (what we have with the multiple checkboxes), the Form automatically creates an array containing each value of the input for the field `name`. If only one checkbox is checked, there's only a single value instead of an array.
If no checkbox is checked, the value for `name` is undefined (and that's the only reason why I need that filter^^ - otherwise I'd create an array containing undefined).

Reply:
I've refactored it a little bit, maybe you can have another look